conanfile_sample_app.py

```diff
@@ -157,8 +158,8 @@ def build(self):
         
         silabs_package_assistant = self.python_requires["silabs_package_assistant"].module
 
-        slcp_files = list(map(str, Path(os.getcwd()).rglob("*.slcp")))
-        slcw_files = list(map(str, Path(os.getcwd()).rglob("*.slcw")))
+        slcp_files = [str(p) for p in Path(os.getcwd()).rglob("*.slcp") if "third_party" not in p.parts]
+        slcw_files = [str(p) for p in Path(os.getcwd()).rglob("*.slcw") if "third_party" not in p.parts]
```

Comment:
This is the same as above at line 112-113. It could be made into helper function